packages/trusted-contribution/src/trusted-contribution.ts

```diff
@@ -23,6 +23,7 @@ interface ConfigurationOptions {
 const WELL_KNOWN_CONFIGURATION_FILE = 'trusted-contribution.yml';
 const DEFAULT_TRUSTED_CONTRIBUTORS = [
   'renovate-bot',
+  'dependabot',
```

Comment:
```suggestion
  'dependabot[bot]',
```

See the actual sender: https://api.github.com/repos/GoogleCloudPlatform/java-docs-samples/pulls/3943